racket/collects/net/http-client.rkt

```diff
@@ -121,34 +129,37 @@
                          #:headers [headers-bs empty]
                          #:content-decode [decodes '(gzip)]
                          #:data [data #f])
+  (define (print-to fmt . args)
+    ;; (eprintf "print-to: ~a~%" (apply format fmt args))
+    (apply fprintf to fmt args))
```

Comment:
As a matter of interest, does the compiler not handle curry well?

Tim

On 09/08/2016 14:59, Jay McCarthy wrote:
> In racket/collects/net/http-client.rkt
> <https://github.com/racket/racket/pull/1411#discussion_r74062556>:
>
>> @@ -121,34 +129,37 @@
>>                           #:headers [headers-bs empty]
>>                           #:content-decode [decodes '(gzip)]
>>                           #:data [data #f])
>> +  (define (print-to fmt . args)
>> +    ;; (eprintf "print-to: ~a~%" (apply format fmt args))
>> +    (apply fprintf to fmt args))
>
> |curry| is very slow too :/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/racket/racket/pull/1411/files/a5583485b67cff28edef92607574f1f9288b836c#r74062556>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABDkIa87_vPdqY8ZnlmSaaff8PKuMJJZks5qeIemgaJpZM4Jf_v6>.
>


-- 
Tim Brown - tim@timb.net - +44(0)7771714159